working-with-components.md

```diff
@@ -0,0 +1,48 @@
+# Working with components
+
+## What are components
+Components can be thought of as sections of user interface on our websites. There are small components, such as buttons and inputs, and larger components such as product tiles and search headers. Typically larger components will be _composed_ of many smaller components. This pattern of designing many small components to compose together for larger reuseable components within a page is known as **atomic design**. To read more on Atomic Design see [Brad Frost's excellent post](http://bradfrost.com/blog/post/atomic-web-design/), but for the purposes of working with components at Hx we'll be referring to three of the 5 atomic design component levels:
+
+ - **Atoms:** These are the basic building blocks of our pages. Atoms are our most simple components, such as a form label, a text input or a button.
+ - **Molecules:** These are groups of atoms bonded together and form larger reuseable components. These molecules take on their own emergent properties and can in turn be reused to build even more complex sections of a user interface. An example of a molecule component could be a label and input together.
+ - **Organisms:** These are groups of molecules joined together to form a relatively complex, distinct sections of our user interface. An example of an organism component may be a product tile, or a search header.
+
+At Holiday Extras we use Facebook's [React](https://facebook.github.io/react/index.html) for building composeable user interface components.
```

Comment:
I think React is the current goal, but we can achieve components and atom design with jump simple template engines like handlebars etc. We should encourage atomic design even on old templating platforms. Also in a few years we would probably have moved on from React (something new will come along)